<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QWidget" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1530</width>
    <height>1052</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Movie Finder</string>
  </property>
  <widget class="QGroupBox" name="resultGroupBox">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>801</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#resultGroupBox {border:0;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>291</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="moviePosterLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>501</width>
      <height>621</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="summaryLabel">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="ratingGroupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#ratingGroupBox {border:0;}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="ratingImdbLogo">
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ratingLabel">
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="imdbLinkLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QtWaitingSpinner" name="spinnerImdb" native="true">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>-100</y>
      <width>351</width>
      <height>741</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>691</width>
     <height>641</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabWidgetPage1">
    <attribute name="title">
     <string notr="true">IMDb</string>
    </attribute>
    <widget class="QGroupBox" name="imdbSearchGroupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>681</width>
       <height>601</height>
      </rect>
     </property>
     <property name="title">
      <string>Search Movie on IMDb</string>
     </property>
     <widget class="QLineEdit" name="searchEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="searchImdbButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QListView" name="imdbResultListView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>661</width>
        <height>501</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="seriesCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Include TV Shows</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage2">
    <attribute name="title">
     <string>Torrent</string>
    </attribute>
    <widget class="QGroupBox" name="torrentSearchGroupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>681</width>
       <height>601</height>
      </rect>
     </property>
     <property name="title">
      <string>First select a movie from IMDb</string>
     </property>
     <widget class="QPushButton" name="torrentSearchButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="jackettCheckBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Jackett API</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="ytsCheckBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>YTS</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListView" name="torrentResultListView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>661</width>
        <height>231</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
     <widget class="QPushButton" name="torrentSendButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>550</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QLabel" name="torrentInfoLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>341</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QtWaitingSpinner" name="spinnerTorrent" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>661</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="torrentUrlLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>500</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="settingsButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>550</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Host Settings</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtWaitingSpinner</class>
   <extends>QWidget</extends>
   <header location="global">waitingspinnerwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
